src/v2/admin/controllers/admController.ts

```diff
@@ -40,6 +40,25 @@ export const banUser = async (req: Request, res: Response) => {
   }
 };
 
+export const createBusiness = async (req: Request, res: Response) => {
+  try {
+    const { name, logo, description } = req.body;
+    await superAdminService.createBusiness(name, logo, description);
+    res.status(200).send('Business created successfully');
+  } catch (error: any) {
```

Comment:
More of a nitpick but we want to avoid the use of any where we can. We want to make sure everything we do has a type.
It helps with debugging and preventing bugs